source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md

```diff
@@ -0,0 +1,174 @@
+```
+[hosts]
+@HOSTNAME@
+
+[disktype]
+@RAIDTYPE@ #Possible values raid6, raid10, raid5, jbod
+
+[diskcount]
+@NUMBER_OF_DATA_DISKS@ #Ignored in case of jbod
+
+[stripesize]
+@STRIPE_SIZE@ #256 in case of jbod
+
+[pv]
+action=create
+devices=@DEVICE@
+
+[vg1]
+action=create
+vgname=RHGS_vg1
+pvname=@DEVICE@
+
+[lv1]
+action=create
+vgname=RHGS_vg1
+lvname=engine_lv
+lvtype=thick
+size=100GB 
+mount=/rhgs/brick1
+
+[lv2]
+action=create
+vgname=RHGS_vg1
+poolname=lvthinpool
+lvtype=thinpool
+poolmetadatasize=15GB
+chunksize=1024k
+size=@SIZE@ #For example: 18000GB, depending on device capacity
+
+[lv3]
+action=create
+lvname=lv_vmdisks
+poolname=lvthinpool
+vgname=RHGS_vg1
+lvtype=thinlv
+mount=/rhgs/brick2
+virtualsize=@SIZE@
+
+[lv4]
+action=create
+lvname=lv_datadisks
+poolname=lvthinpool
+vgname=RHGS_vg1
+lvtype=thinlv
+mount=/rhgs/brick3
+virtualsize=@SIZE@
+
+#[vg2]
+#action=extend
+#vgname=RHGS_vg1
+#pvname=@SSD_DEVICE@
+
+#[lv5]
+#action=setup-cache
+#ssd=@SSD_DEVICE@
+#vgname=RHGS_vg1
+#poolname=lvthinpool
+#cache_lv=lvcache
+#cache_lvsize=5GB # Provide device size
+## cachemode=writeback
+
+[selinux]
+yes
+
+[service3]
+action=restart
+service=glusterd
+slice_setup=yes
+
+[firewalld]
+action=add
+ports=111/tcp,2049/tcp,54321/tcp,5900/tcp,5900-6923/tcp,5666/tcp,16514/tcp,54322/tcp
+services=glusterfs
+
+[script2]
+action=execute
+file=/usr/share/gdeploy/scripts/disable-gluster-hooks.sh
+
+[volume]
+action=create
+volname=engine
+transport=tcp
+key=storage.owner-uid,storage.owner-gid,features.shard,performance.low-prio-threads,performance.strict-o-direct,network.remote-dio,network.ping-timeout,user.cifs,nfs.disable,performance.quick-read,performance.read-ahead,performance.io-cache,cluster.eager-lock
+value=36,36,on,32,on,off,30,off,on,off,off,off,enable
+brick_dirs=/rhgs/brick1/engine
+ignore_volume_errors=no
+
+[volume2]
+action=create
+volname=vmstore
+transport=tcp
+key=storage.owner-uid,storage.owner-gid,features.shard,performance.low-prio-threads,performance.strict-o-direct,network.remote-dio,network.ping-timeout,user.cifs,nfs.disable,performance.quick-read,performance.read-ahead,performance.io-cache,cluster.eager-lock
+value=36,36,on,32,on,off,30,off,on,off,off,off,enable
+brick_dirs=/rhgs/brick2/vmstore
+ignore_volume_errors=no
+
+[volume3]
+action=create
+volname=data
+transport=tcp
+key=storage.owner-uid,storage.owner-gid,features.shard,performance.low-prio-threads,performance.strict-o-direct,network.remote-dio,network.ping-timeout,user.cifs,nfs.disable,performance.quick-read,performance.read-ahead,performance.io-cache,cluster.eager-lock
+value=36,36,on,32,on,off,30,off,on,off,off,off,enable
+brick_dirs=/rhgs/brick3/data
+ignore_volume_errors=no
+```
+
+Once the configuration file is edited (the @ @ replaced), the gluster environment can be setup using
+```
+gdeploy -c gdeploy.conf
```

Comment:
Why are we not promoting the Cockpit-based installation?

Reply:
For single node - we donot have the Cockpit based UI.